docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: "3.9"
+
+services:
+  postgres:
+    image: postgres:10-bullseye
+    ports:
+      - "5433:5432"
+    environment:
+      POSTGRES_USER: pythondotorg
+      POSTGRES_PASSWORD: pythondotorg
+      POSTGRES_DB: pythondotorg
+      POSTGRES_HOST_AUTH_METHOD: trust  # never do this in production!
+    healthcheck:
+      test: ["CMD", "pg_isready", "-U", "pythondotorg", "-d", "pythondotorg"]
+      interval: 1s
+
+  web:
+    build: .
+    command: sh dev/wait-for.sh postgres:5432 -- dev/docker-entrypoint.sh python manage.py runserver 0.0.0.0:8000
```

Comment:
Ah! So the `wait-for.sh` and `entrypoint.sh` scripts won't be necessary for this! We're replacing `wait-for.sh` with the health check configured for Postgres and the depends_on configuration for web.

Additionally we'll discuss how we're replacing the entrypoint  with Makefile targets as well.

The command we run for the web container can probably just be `python manage.py runserver 0.0.0.0:8000`.